python/gink/tests/test_graph.py

```diff
@@ -27,33 +27,34 @@ def test_basics():
             assert edges == {edge2}
 
 def test_to_from():
-    store = LmdbStore()
-    db = Database(store=store)
-    bundler = Bundler()
-    vertex1 = Vertex(bundler=bundler)
-    vertex2 = Vertex(bundler=bundler)
-    db.bundle(bundler=bundler)
-    edge_type = EdgeType()
-    bundler = Bundler()
-    edge12 = edge_type.create_edge(vertex1, vertex2, bundler=bundler)
-    edge21 = edge_type.create_edge(vertex2, vertex1, bundler=bundler)
-    db.bundle(bundler=bundler)
-    edges_from1 = set(vertex1.get_edges_from())
-    assert edges_from1 == {edge12}, edges_from1
-    edges_to = set(vertex1.get_edges_to())
-    assert edges_to == {edge21}, edges_to
+    for store in [LmdbStore()]:
```

Comment:
Did you want to include MemoryStore here?

Reply:
MemoryStore doesn't support getting edges with a source or target yet. That is going to be one of the next things I work on. While I changed the other tests I figured I'd just make them all look similar now.